src/components/jobs/JobsList.jsx

```diff
@@ -28,40 +29,26 @@ const displayJobs = (studiesFilter, typesFilter, jobs) => {
 
 const createJobs = jobbies => {
   return jobbies.map(job => (
-    <JobItem
-      className="item"
-      key={job.node.id}
-      job={job.node}
-      partner={job.node.partner}
-    />
+    <Grid.Column>
+      <JobItem key={job.node.id} job={job.node} partner={job.node.partner} />
+    </Grid.Column>
   ));
 };
 
 const jobslist = props => (
   <JobsList>
     <div className="container">
-      {displayJobs(props.studiesFilter, props.typesFilter, props.jobs)}
+      <Grid doubling stackable stretched columns={3}>
```

Comment:
Remove `stackable`. `doubling` includes that functionality.

Reply:
No it does not. Check the mobile view :). Removing stackable won't allow the cards the stack 1 on 1.